README.md

```diff
@@ -7,6 +7,8 @@ https://gitforwindows.org/
 
 ## Developing
 
+Note: if you have the 'separate git bash and cmd window' setting then you may need to change from bash to cmd at step 2, adjusting the path at step 3.
+
```

Comment:
I guess the actual fix would be to reorder "Install Node.js" to be the first item, with a suggestion to add it to the `PATH`, and to (re-)open the terminal after the installation so that `node` and `npm` are in the `PATH`.

The comment above might better read as "For this to work, either use Git Bash, or make sure that the (recommend) "Git from the command-line" option is used in Git for Windows' installer page titled "Adjusting your PATH environment".

Reply:
Oh, and I guess while you're at it, remove the last line that reads "(Tested on XP, and onwards)". It does not really make sense to keep a comment like that, as it is prone to become stale.

Reply:
the screen shot shows that we don't yet do the "(recommend)" part of the 'Git from the command-line' option...

It is a tricky distinction between 'safe' and 'recommended'.. Thoughts?

Reply:
I think we should change the description.

![image](https://user-images.githubusercontent.com/127790/53639767-6dcdaf00-3c2a-11e9-87e6-e55a07864eac.png)

I should have said "default" when I said "recommended", but you're of course right that we should make it "recommended" because it is already the default.

Maybe another thing you could help with?

Reply:
I'll have a look. Just need to find the bit of code that generates that screen shot and add the "(Recommended)" to the option line, so that the hold-out will see that they should make the step ;-)

Reply:
https://github.com/git-for-windows/build-extra/blob/a5a0286f0d7fda247da11a9618c839b4913bbde3/installer/install.iss#L1647-L1660

Reply:
Test using the Git for Windows SDK, call `sdk cd build-extra`, then edit `installer/install.iss` and test with `./installer/release.sh -d Path`.

Reply:
> test with `./installer/release.sh -d Path`.

(This will obviously *not* install anything, just show you the wizard page.)

Reply:
minor glitch with the sdk command - were you expecting the 'fatal: ref HEAD is not a symbolic ref' message? I'm ignoring it at the moment ;-)  Though I'll have to step onto a specific branch - not sure why/when it went detached!

```
phili@Philip-Win10 MINGW64 /usr/src/git-for-windows.github.io (snapshots)
$ sdk cd build-extra
fatal: ref HEAD is not a symbolic ref

phili@Philip-Win10 MINGW64 /usr/src/build-extra ((c2f66e1...))
$ sdk cd build-extra
fatal: ref HEAD is not a symbolic ref

phili@Philip-Win10 MINGW64 /usr/src/build-extra ((c2f66e1...))
$ cd /

phili@Philip-Win10 MINGW64 / (master)
$ sdk cd build-extra
fatal: ref HEAD is not a symbolic ref

phili@Philip-Win10 MINGW64 /usr/src/build-extra ((c2f66e1...))
$
```

Reply:
> fatal: ref HEAD is not a symbolic ref

That's most likely due to https://github.com/git-for-windows/build-extra/blob/a5a0286f0d7fda247da11a9618c839b4913bbde3/git-extra/git-sdk.sh#L159 needing to be redirected (`>/dev/null 2>&1` just after the `HEAD`).

If you could test this, by editing `/etc/profile.d/git-sdk.sh` and then calling `sdk reload` before calling `sdk cd build-extra` again?